include/znc/User.h

```diff
@@ -127,7 +127,9 @@ class CUser {
 	void SetQuitMsg(const CString& s);
 	bool AddCTCPReply(const CString& sCTCP, const CString& sReply);
 	bool DelCTCPReply(const CString& sCTCP);
-	bool SetBufferCount(unsigned int u, bool bForce = false);
+	bool SetBufferCount(unsigned int u, bool bForce = false); // deprecated
```

Comment:
Deprecated, but what does it do now?
Also please move this //deprecated to doxygen, otherwise it won't be visible on http://docs.znc.in